src/dataviews/dataview-model-base.js

```diff
@@ -322,6 +322,10 @@ module.exports = Model.extend({
     return sourceAnalysis && sourceAnalysis.get('type');
   },
 
+  getLayerName: function () {
+    return this.layer.get('layer_name');
```

Comment:
awesome, in case there's no layer I have that case covered https://github.com/CartoDB/deep-insights.js/pull/552/files#diff-0c6eb42296959f824ed2b8c1e7927e94R39